src/lambda/commands/deploySamApplication.ts

```diff
@@ -324,3 +330,36 @@ function getDefaultWindowFunctions(): WindowFunctions {
         showInformationMessage: vscode.window.showInformationMessage,
     }
 }
+
+function notifyDuringLongDeployment(deployParameters: DeploySamApplicationParameters): NodeJS.Timeout {
+    return setTimeout(async () => {
+        const cfnClient = new DefaultCloudFormationClient(deployParameters.region)
+        const stack = (await cfnClient.describeStacks(deployParameters.destinationStackName)).Stacks
+        if (stack !== undefined) {
+            const helpLink = localize('AWS.samcli.deploy.stuckStackHelpLink', 'View docs')
```

Comment:
vscode commands (and menu items) use Title Case. however maybe this should just say "Documentation"

```suggestion
            const helpLink = localize('AWS.samcli.deploy.stuckStackHelpLink', 'Documentation')
```